python_modules/dagster/dagster/_core/definitions/asset_daemon_context.py

```diff
@@ -428,15 +432,41 @@ def evaluate_asset(
                 dynamic_partitions_store=self.instance_queryer,
             ),
             to_materialize,
+            to_skip,
             to_discard,
         )
 
+    @functools.cached_property
+    def skipped_asset_graph_subset(self) -> AssetGraphSubset:
```

Comment:
This means the subset that was skipped on the last tick, right? A name like `skipped_on_last_tick_asset_graph_subset` would make this more clear.

Do we typically use `_asset_graph_subset` on methods that return `AssetGraphSubset`s? Taking that out might be a way to make the name less verbose.